lib/socks.c

```diff
@@ -637,7 +637,6 @@ CURLcode Curl_SOCKS5(const char *proxy_user,
             "Undocumented SOCKS5 mode attempted to be used by server.");
       return CURLE_COULDNT_CONNECT;
     }
-    break;
```

Comment:
I think this code will be somewhat clearer if we also move this closing brace up three lines to be before the final `failf()` call there. What do you think?